Makefile

```diff
@@ -27,6 +27,10 @@ ISTIO_DOCKER_HUB ?= docker.io/istio
 export ISTIO_DOCKER_HUB
 ISTIO_GCS ?= istio-release/releases/$(VERSION)
 ISTIO_URL ?= https://storage.googleapis.com/$(ISTIO_GCS)
+ISTIO_CNI_DOCKER_HUB ?= docker.io/istio
```

Comment:
why not just use ISITO_DOCKER_HUB for this variables?

Reply:
see below

Reply:
Since cni is being added to release, I don't suppose we need a separate variable for CNI hub and tag here?

For e2e tests we can use different cni hub and tag (like https://github.com/istio/istio/blob/9b8397dccbf078a4fbe23d8e99d5c57475ecc43b/tests/istio.mk#L60) but I don't suppose we need this in the top level Makefile.